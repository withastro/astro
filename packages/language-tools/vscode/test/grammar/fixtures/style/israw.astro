<style is:raw>
	h1 {
		color: red
	}
</style>
