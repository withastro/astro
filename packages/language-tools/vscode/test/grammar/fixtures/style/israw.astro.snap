><style is:raw>
#^ source.astro meta.scope.tag.style.astro meta.raw.astro meta.tag.start.astro punctuation.definition.tag.begin.astro
# ^^^^^ source.astro meta.scope.tag.style.astro meta.raw.astro meta.tag.start.astro entity.name.tag.astro
#      ^ source.astro meta.scope.tag.style.astro meta.raw.astro source.unknown meta.tag.start.astro
#       ^^^^^^ source.astro meta.scope.tag.style.astro meta.raw.astro source.unknown meta.tag.start.astro meta.attribute.is:raw.astro entity.other.attribute-name.astro
#             ^ source.astro meta.scope.tag.style.astro meta.raw.astro source.unknown meta.tag.start.astro punctuation.definition.tag.end.astro
>	h1 {
#^^^^^^ source.astro meta.scope.tag.style.astro meta.raw.astro source.unknown
>		color: red
#^^^^^^^^^^^^^ source.astro meta.scope.tag.style.astro meta.raw.astro source.unknown
>	}
#^^^ source.astro meta.scope.tag.style.astro meta.raw.astro source.unknown
></style>
#^^ source.astro meta.scope.tag.style.astro meta.raw.astro meta.tag.end.astro punctuation.definition.tag.begin.astro
#  ^^^^^ source.astro meta.scope.tag.style.astro meta.raw.astro meta.tag.end.astro entity.name.tag.astro
#       ^ source.astro meta.scope.tag.style.astro meta.raw.astro meta.tag.end.astro punctuation.definition.tag.end.astro
>